packages/terra-functional-testing/tests/jest/services/wdio-terra-service.test.js

```diff
@@ -1,17 +1,11 @@
 const TerraService = require('../../../src/services/wdio-terra-service');
 
 describe('WDIO Terra Service', () => {
-  it('should add the axe command to the browser object', () => {
```

Comment:
Should we validate the expect helper is also available?

Reply:
I haven't found a good way to use `expect` to mock `expect` and verify a custom `expect` is available if that makes sense. This gets tested indirectly via WDIO. I tried using `jest.spyOn(expect, 'extend')` but it didn't return the results I was expecting. I can probably mock axe and call the helper directly, but I don't know what that would really be testing. Do you have any ideas for a way to test this in jest?